tests/regression/test_tensor_elements.py

```diff
@@ -0,0 +1,51 @@
+from firedrake import *
+import pytest
+
+
+rg = RandomGenerator(PCG64(seed=1))
+
+
+@pytest.fixture(params=("square", "cube"))
+def mesh(request):
+    if request.param == "square":
+        return UnitSquareMesh(2, 2)
+    elif request.param == "cube":
+        return UnitCubeMesh(2, 2, 2)
+    else:
+        raise ValueError("Unrecognized mesh type")
+
+
+@pytest.mark.parametrize("family, degree", [
```

Comment:
JM is a tensor element.  Does it want to be tested here as well?

Reply:
Not really, here we are only testing the distributional tensor-valued elements. JM, AW, and HZ are tested in test_stress_elements.py